next-desolates-map/components/SettingsPage/SettingsPage.tsx

```diff
@@ -0,0 +1,466 @@
+import { Clusters, SpaceRendererContext, Stages } from "../../pages/_app";
+import { searchForPlanetAndChangeStage } from "../RenderController/RenderController";
+import Image from "next/image"
+import pic from "../../public/profile.png"
+
+
+export default function SettingsPage() {
+    const {
+        changeStage,
+        p5,
+        cam,
+        setOngoingCamMov,
+        celestialObjects,
+        setCluster,
+        setSelectedPlanet,
+    } = useContext(SpaceRendererContext);
+
+    const [planetSearchInput, setPlanetSearchInput] = useState("");
+
+    function handleInputChange(e: any) {
+        setPlanetSearchInput(e.target.value);
+    }
+
+    function handleKeyDown(e: any) {
+        if (e.key === "Enter") {
+            console.log("ENTER PRESSED");
+            searchForPlanet();
+        }
+    }
+
+    function searchForPlanet() {
+        searchForPlanetAndChangeStage(
+            planetSearchInput,
+            p5!,
+            cam!,
+            setOngoingCamMov,
+            celestialObjects,
+            setCluster,
+            changeStage,
+            setSelectedPlanet
+        );
+    }
+
+    return (
+        // Global window
+     <div className="fixed top-7 right-0 left-0 flex mt-10 lg:absolute lg:flex lg:justify-center">
+        <div className="fixed ml-5 mr-5 rounded-lg backdrop-filter backdrop-blur outline-cool shadow-cool bg-faded max-w-[1100px] absolute top-0 right-0 flex flex-col max-h-180 p-2 overflow-y-auto select-none no-scrollbar lg:relative lg:max-w-[900px] lg:top-0 lg:flex lg:flex-col lg:py-0 lg:mt-5 lg:overflow-hidden lg:h-max">
+            
+            <div className="pb-1 flex items-center lg:pt-0 lg:pb-0">
+
+                <div className="flex-1 max-w-4x1 mx-auto p-2 lg:p-6">
+
+                {/* Section 1 - header */}
+
+                <section>
+                    <div className="hidden lg:flex lg:justify-between">
+                        <p className="text-lg font-cool text-primary">Profile</p>
+                        <p className="text-lg font-cool text-primary">Owner</p>
+                    </div>
+                    <div className="hidden lg:flex lg:justify-end">
+                        <p className="text-lg font-cool text-primary">1453..3178</p>
+                    </div>
+                </section>
+
+                {/* Section 1 - body */}
+
+                <ul className="max-h-screen justify-around lg:flex lg:justify-start">
+                    <div className="lg:flex lg:basis-1/4 lg:items-center">
+                        
+                        {/* Profile picture */}
+                        <li className="">
+                            <div className="flex justify-center pt-2 pb-2 lg:flex lg:basis-1/8 lg:mr-10">
+                                <Image
+                                    src={pic}
+                                    alt="Picture of the author"
+                                    width={120}
+                                    height={120}
```

Comment:
It's better to use tailwind classes for width and height, than using these two attributes

you can play with some of the avalable ones here -> https://tailwindcss.com/docs/width

Reply:
Done.